GOSS_tests_for_sbps.md

```diff
@@ -0,0 +1,48 @@
+# Assumptions before running this test: 
+
+Ansible scripts are run successfully which configures iSCSI on selected worker nodes and label them as 'iscsi=sbps'. Goss tests use this label as identification of nodes on which this test has to run. 
+
+## SBPS GOSS Frame work:
+
+SBPS GOSS tests run some of the sanity tests for SBPS solution and these tests run on worker node(s) where iSCSI is configured and SBPS Marshal agent is running. Bash script 'https://github.com/Cray-HPE/csm-testing/blob/release/1.6/goss-testing/scripts/iscsi_sbps_sanity.sh' does the below mentioned sanity checksfor SBPS:
+
+- Verify SPBS Agent is running (systemd service)
+- Verify LIO target is active
+- Verify TCP service probes complete against all active portals (iSCSI)
+- Verify DNS SRV and A records exist for the worker respective of the iSCSI portals
+- Mapping between DNS A records and host iscsi portals

```

Comment:
Should this be "**Verify the** mapping ..."?

Reply:
Yes, incorporated. Thanks.